### PR TITLE
WIP: {phys}[foss/2020b,intel/2020b] ASAP3 v3.12.3 w/ Python 3.8.6

### PR DESCRIPTION
**Pull request by schiotz**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by schiotz** [comment]
This is a bug-fix release of Asap, fixing a memory leak introduced in version 3.12.1.

File: easybuild/easyconfigs/a/ASAP3/ASAP3-3.12.3-foss-2020b-ASE-3.21.1.eb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+versionsuffix = '-ASE-' + _aseversion
+
+homepage = 'https://wiki.fysik.dtu.dk/asap/'
+description = """ASAP is a calculator for doing large-scale classical molecular
+dynamics within the Campos Atomic Simulation Environment (ASE)."""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+toolchainopts = {'pic': True, 'usempi': True, 'openmp': False}
+
+source_urls = [PYPI_LOWER_SOURCE]
```
Comment (smoors) [inline easybuild/easyconfigs/a/ASAP3/ASAP3-3.12.3-foss-2020b-ASE-3.21.1.eb:15]: please remove this line, it is now the default for PythonPackages and PythonBundles.

File: easybuild/easyconfigs/a/ASAP3/ASAP3-3.12.3-intel-2020b-ASE-3.21.1.eb [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+versionsuffix = '-ASE-' + _aseversion
+
+homepage = 'https://wiki.fysik.dtu.dk/asap/'
+description = """ASAP is a calculator for doing large-scale classical molecular
+dynamics within the Campos Atomic Simulation Environment (ASE)."""
+
+toolchain = {'name': 'intel', 'version': '2020b'}
+toolchainopts = {'pic': True, 'usempi': True, 'openmp': False}
+
+source_urls = [PYPI_LOWER_SOURCE]
```
Comment (smoors) [inline easybuild/easyconfigs/a/ASAP3/ASAP3-3.12.3-intel-2020b-ASE-3.21.1.eb:15]: same here.

**Review by smoors — CHANGES_REQUESTED** [review]

**Comment by schiotz** [comment]
@smoors 
Thank you for your feedback.  I have stalled on this one because I found a couple of bugs in Asap, and I wanted to squash them first - one of them was a bit tricky.  I expect to release a new version tomorrow or Wednesday, and then I will use the feedback of this one to make a new PR with the newest version.

**Comment by schiotz** [comment]
Withdrawn.  This PR is made obsolete by #12703.

**Pull request closed without merging**